omegaconf/base.py

```diff
@@ -615,3 +615,9 @@ def _invalidate_flags_cache(self) -> None:
 
     def _has_ref_type(self) -> bool:
         return self._metadata.ref_type is not Any
+
+
+class SCMode(Enum):
+    DICT = 1  # convert to plain dict
+    DICT_CONFIG = 2  # Keep as OmegaConf DictConfig
+    INSTANTIATE = 3  # instantiate underlying object.
```

Comment:
Let's keep instantiate out for now.
```suggestion
class SCMode(Enum):
    DICT = 1  # convert to plain dict
    DICT_CONFIG = 2  # Keep as OmegaConf DictConfig
```